src/cli/restore/exchange.go

```diff
@@ -106,7 +82,11 @@ func restoreExchangeCmd(cmd *cobra.Command, args []string) error {
 		return nil
 	}
 
-	opts := getRestoreExchangeCmdOpts(cmd)
+	opts := utils.MakeExchangeOpts(cmd)
+
+	if utils.RunMode == utils.RunModeFlagTest {
```

Comment:
These tests were not originally intended to execute the command.  They're primarily supposed to catch command construction risks.  This flag, while hacky, isn't affecting our test coverage in any important ways.  And dependency injection via cobra commands isn't supported, and would likely end up equally hacky.

Still, I'm not a fan of test-only handling in the first place.  A more resilient implementation would be to move the options construction into a pre-run hook and pass it along to the primary handler.  That way the unit tests could call the pre-run func in the command instead of executing the command itself.  A more healthy flow all around.